src/widgets/index.js

```diff
@@ -19,4 +20,5 @@ export default {
   topicFrequencyGraph,
   targetPopulationTable,
   frequencyGraph,
+  goalStatusGraph,
```

Comment:
Probably not a huge deal, but querying this API returns 238 goals with a status of "" on my DB. Possibly something weird from imported data?

![Screen Shot 2022-02-09 at 7 59 19 AM](https://user-images.githubusercontent.com/3288586/153206037-12f01b95-653f-4410-9106-a5457ceb4413.png)

Reply:
Yeah those goals come in from smartsheet as empty strings. I'd imagine we'd want to update those goals to have a status that matches "needs status" (either null or explicitly "needs status")